### PR TITLE
Added maven. Added tests for Prime and Car

### PR DESCRIPTION
**Pull request by DenisLobur**

### CODE REVIEW

File: src/test/java/lesson6/CarTest.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
+package lesson6;
+
+import lesson6.Car;
+import org.junit.jupiter.api.*;
+
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+public class CarTest {
+
+    Car honda;
```
Comment (nikmikhailov13) [inline src/test/java/lesson6/CarTest.java]: These fields could be private

File: src/test/java/lesson5/PrimeTest.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
+package lesson5;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+class PrimeTest {
+
+    @Test
+    void checkIfTenPrime() {
+        boolean ten = Prime.isPrime(10, 2);
```
Comment (nikmikhailov13) [inline src/test/java/lesson5/PrimeTest.java]: Better to call a boolean variable `is... `
For example `isPrime`

File: src/main/java/lesson6/Car.java [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    private double fuelPerHundredKm;
+    @Getter
+    private long lastInspectionTime;
+
+    public boolean canRide() {
+        return tankCapacity > 0.0;
+    }
+
+    public boolean isOperational() {
+        long twoYearsInMillis = 2 * 31536000000L;
```
Comment (nikmikhailov13) [inline src/main/java/lesson6/Car.java]: Why did you convert it to millis? 😄 
Even if you do so, you could have used already multiplied number `63072000000L`

**Review by nikmikhailov13 — APPROVED** [review]
LGTM
Minor comments

File: src/main/java/lesson6/Car.java (hunk 3 above)
Comment (DenisLobur) [reply src/main/java/lesson6/Car.java]: To use it with System.currentTimeMillis which is in millis

**Pull request merged**